web/src/features/fba/components/viz/FuelSummary.tsx

```diff
@@ -0,0 +1,130 @@
+import React, { useEffect, useState } from 'react'
+import { FireShape, FireZoneThresholdFuelTypeArea } from 'api/fbaAPI'
+import { Box, Tooltip, Typography } from '@mui/material'
+import { groupBy, isUndefined } from 'lodash'
+import { DateTime } from 'luxon'
+import FuelDistribution from 'features/fba/components/viz/FuelDistribution'
+import { DataGridPro, GridColDef, GridRenderCellParams } from '@mui/x-data-grid-pro'
+import { useTheme } from '@mui/material/styles'
+
+export interface FuelTypeInfoSummary {
+  area: number
+  criticalHoursStart?: DateTime
+  criticalHoursStop?: DateTime
```

Comment:
Minor nit: we use "start" and "end" elsewhere

```suggestion
  criticalHoursEnd?: DateTime
```

Reply:
Fixed in https://github.com/bcgov/wps/pull/3792/commits/2025fad89a3a04d99b7c15d2956bb9b31b947584